annotations/openapi/oas-annotations.raml

```diff
@@ -0,0 +1,30 @@
+#%RAML 1.0 Library
+
+usage: |
+  This library contains a set of standard OpenAPI related annotations that are not supported in RAML.
+
+types:
+  externalDocs:
+    properties:
+      description?: string
+      url: string
+  
+annotationTypes: 
+  oas-summary: !include oas-summary.raml
+  oas-deprecated: !include oas-deprecated.raml
```

Comment:
As for me it is more logical to create generic deprecated annotation, which is not specific to oas.